flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkInputFormat.java

```diff
@@ -33,24 +33,20 @@
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.catalog.TableIdentifier;
-import org.apache.iceberg.data.GenericAppenderHelper;
+import org.apache.iceberg.data.GenAppenderHelper;
 import org.apache.iceberg.data.RandomGenericData;
 import org.apache.iceberg.data.Record;
 import org.apache.iceberg.flink.FlinkSchemaUtil;
 import org.apache.iceberg.flink.TestFixtures;
 import org.apache.iceberg.flink.TestHelpers;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 import org.apache.iceberg.types.Types;
-import org.junit.Assume;
-import org.junit.Test;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
 /** Test {@link FlinkInputFormat}. */
-public class TestFlinkInputFormat extends TestFlinkSource {
-
-  public TestFlinkInputFormat(String fileFormat) {
-    super(fileFormat);
-  }
-
+public class TestFlinkInputFormat extends TestFlinkSrc {
```

Comment:
How many tests inherit from the `TestFlinkSource`?
Would it be too much to review in one run? Or would it be easier to move all of them at once.
If we move all of them at once, we would not need `TestFlinkSrc`, and we can keep, and migrate `TestFlinkSource` instead

Reply:
I can see the following classes impacted by `TestFlinkSource`:
- TestFlinkScanSql

In `TestFlinkScan`, there's `HadoopCatalogResource` which is used by quite a number of unit tests. I had renamed `HadoopCatalogResource` to `HadoopCatResource` in this commit and migrate the impacted classes listed above.